superset-frontend/src/explore/components/controls/FixedOrMetricControl/index.jsx

```diff
@@ -22,10 +22,10 @@ import { css } from '@emotion/core';
 import { t } from '@superset-ui/core';
 import Label from 'src/components/Label';
 import Collapse from 'src/common/components/Collapse';
-import TextControl from './TextControl';
-import MetricsControl from './MetricControl/MetricsControl';
-import ControlHeader from '../ControlHeader';
-import PopoverSection from '../../../components/PopoverSection';
+import TextControl from '../TextControl';
+import MetricsControl from '../MetricControl/MetricsControl';
+import ControlHeader from '../../ControlHeader';
+import PopoverSection from '../../../../components/PopoverSection';
```

Comment:
src path might be a good idea here for these ones that are further up the hierarchy

Reply:
Fixed.